scripts/runDockerFile.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker run local/clang-format-action:latest $@
```

Comment:
The used shell-linter complains about `$@` potentially being re-split - but this is actually exactly what we want in this case, since we want to pass the parameters separately and not as one big, concatenated chunk